### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by TWiStErRob**

### CODE REVIEW

**Review by Goooler — APPROVED** [review]

File: src/compatTest/kotlin/io/github/gradlenexus/publishplugin/BaseNexusPublishPluginTests.kt [hunk 1]
```diff
@@ -1037,15 +1031,10 @@ abstract class BaseNexusPublishPluginTests {
         gradleRunner(*arguments).buildAndFail()
 
     private fun gradleRunner(vararg arguments: String): GradleRunner {
-        val warnings = when {
-            // Failing only became an option at Gradle 5.6.
-            gradleVersion >= GradleVersion.version("5.6") -> "fail"
-            else -> "all"
```
Comment (szpak) [inline src/compatTest/kotlin/io/github/gradlenexus/publishplugin/BaseNexusPublishPluginTests.kt:1043]: Good catch.

**Review by szpak — APPROVED** [review]
Thanks. I plan to merge it after #242 and #246.

**Comment by szpak** [comment]
@TWiStErRob Could you rebase it against master (to apply the fix for testing)?

**Comment by TWiStErRob** [comment]
Done.

Tip: if you enable this in the repo Settings:
image
contributors and you will get this button:
image

**Comment by szpak** [comment]
> If you enable this in the repo Settings:

Nice, I didn't know.

**Pull request merged**